workloads/templates/common-templates.md

```diff
@@ -131,3 +131,46 @@ oc process rhel7-generic-tiny PVCNAME=mydisk NAME=rheltinyvm | oc apply -f -
 
 Please note that, after the generation step, VM objects and template objects have no relationship with each other besides the aforementioned label (e.g. changes
 in templates do not automatically affect VMs, or vice versa).
+
+## common template customization
+
+### template customization - memory, CPU
+
+There are three options to customize VM memory and CPU:
+
+ * select flavor - tiny, small, medium, etc. Each flavor grants to VM different amount of RAM and CPU cores.
+ * setting up directly editable fields spec.template.spec.domain.cpu.cores and spec.template.spec.domain.resources.requests.memory, example:
+
+```bash
+oc patch virtualmachine testguest --type merge -p '{"spec":{"template":{"spec":{"domain":{"cpu":{"cores":'3'}}}}}}'
+virtualmachine.kubevirt.io/testguest patched
+```
+ * use WebUI: WebUI supports different flavors and Workload Profiles. Also it supports custom amount of memory and CPUs
+
+### template customization - networking
+
+There are two options to customize Networking in VM:
+ * setting up directly editable field(see example above): spec.template.spec.networks. You can edit following:
+   * name (of interface)
+   * pod
```

Comment:
You can't edit them if the editable fields do not list them. Each template can have a different list of editable fields..
Also couple of examples would be nice.